code-of-conduct/guide-code-of-conduct-response-plan.md

```diff
@@ -0,0 +1,188 @@
+You can customize the provided boilerplate text or write your own.
+
+
+### About the "Requirements for community moderators" section
+
+In this section, explain the eligibility requirements for your community moderators.
+Once again, this section will require some pre-work for you to think through what your community moderators will need to do to be prepared to serve in this role:
+
+- How should individuals be selected or nominated to serve in this role?
+- What qualifications should someone in this role have?
+- What training will be required to serve in this role?
+
+Community moderators do need training in handling Code of Conduct incidents in order to be effective in this role.
+
+Some training options and resources you can consider using:
+
+- **[The Code of Conduct book](https://frameshiftconsulting.com/resources/code-of-conduct-book/)** - This free book can act as a baseline guide for handling Code of Conduct incidents. Your community moderator team could possibly host a book club and go through the chapters together, discussing them as they go. The authors of this book also offer Code of Conduct training workshops.
+- **[Mozilla's Community Participation Guidelines Enforcement](https://mozilla.teachable.com/p/cpg-training-contributors)** - This free course was developed by Mozilla for Mozilla community moderators and only takes a few hours to complete. However, be aware that this course is not as comprehensive as your community may need. The training is also specific to Mozilla's Code of Conduct, although some of its principles are general enough to apply to many projects.
+- **[Otter Tech Code of Conduct Enforcement Training](https://otter.technology/code-of-conduct-training/)** - Otter Tech is a diversity and inclusion consulting firm that offers regular workshops. The course provides a general framework for how to respond to Code of Conduct violations and provides many opportunities to role-play and practice acting as a community moderator. The Otter Tech training is excellent, but expensive and is only offered in U.S. friendly time zones.
+- If there are other resources that we are not aware of, please (open an issue)[https://github.com/thegooddocsproject/templates/issues] to let us know!
+
+In addition to these resources, we recommend holding regular practice sessions with your community moderators where you role-play what it would be like to handle an actual incident if one were to occur.
+Try creating a hypothetical situation and seeing if you can take it through the entire process to its conclusion.
+
+
+### About the "Community moderator terms of service" section
+
+In this section, describe your plan for rotating community moderators or limiting terms of service if needed.
+
+
+### About the "Reviewing the Code of Conduct" section
+
+In the *Reviewing the Code of Conduct* section, indicate your timetable for reviewing the Code of Conduct and supporting policy documents.

```

Comment:
query: sometimes we say "In this section" and sometimes we put the name in italics. Is there a style? We should be consistent throughout the document set. I'm in two minds about which one I prefer.

Reply:
And here I thought that I was avoiding monotony by changing it up from time to time. :)